README.md

```diff
@@ -55,10 +60,10 @@ There are a few things I know I could be better at. This is not an exhaustive li
 * I am working on understanding introverts. I generally come up ENFJ on Myers-Briggs scales. I am highly aware that engineers are rarely extroverts. I thrive and gain energy from talking to people, and while I know that this is not the case for introverts I sometimes need to be reminded.
```

Comment:
```suggestion
* I am working on understanding introverts. I generally come up ENFJ on Myers-Briggs scales. I am highly aware that engineers are rarely extroverts. I thrive and gain energy from talking to people and while I know that this is not the case for introverts, I sometimes need to be reminded.
```

Reply:
This one is still a run-on, I'll address in a follow up